### DISCUSSION TITLE
Get your free license key form

### DISCUSSION THREAD

**Discussion by jgarec** [mend/renovate-ce-ee#302] [Q&A] [answered]
Hello,

Is there an issue with this form ? It fails with an error when trying to check my email : this email cannot be submitted 

Under the hood, there is a 500 http error with https://www.mend.io/wp-json/my/v1/verify-email

Thx for your feedback

**Comment by justo-mend** [comment]
Thanks for raising the issue. I have verified the same.
When trying to submit the registration form, the client-side form validation shows an error on the email address field - even when a valid email is entered.
The error reads: "`This email cannot be submitted.`"
![image](https://github.com/mend/renovate-on-prem/assets/122523970/3e692e54-fa54-4577-a2c3-fa31e8120e24)

Fix is being worked on.

**Comment by justo-mend** [comment, marked as answer]
Issue is fixed. Please try again. Sorry for the inconvenience.
